fb_instant_articles.module

```diff
@@ -18,6 +18,16 @@ function fb_instant_articles_menu() {
     'file path' => drupal_get_path('module', 'system'),
   );
 
+  $items['admin/config/services/fb-instant-articles/settings'] = array(
```

Comment:
@scottrigby No, the Configure link on modules page leads to `admin/config/services/fb-instant-articles` which contains navigation to the config pages of the sub-modules and I think is what we want, right? But if I hit `admin/config/services/fb-instant-articles/settings` I don't get the settings form, but just that same navigation page.